app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/fragments/ContactStatsFragment.java

```diff
@@ -65,9 +64,15 @@
     private static final int DEC = 12;
     private static final int DEFAULT_VALUE = 0;
     private final String TAG = "sms";
-
     int highestValue;
+    /*
+    These color constants are hardcoded for backwards compatibility.
```

Comment:
Move the comment to be closer to the colors.

Also why? "for backwards compatibility" doesn't tell me what the problem is